CONTRIBUTING.md

```diff
@@ -130,6 +131,25 @@ when creating documentation. Refer to our [Component Guide](COMPONENT_GUIDE.md)
 4. If you are willing to assist in the process of creating a new page, then follow the instructions above to `Fork` or `Clone` the repo and make your edits.
 5. Follow the instructions above to submit a `PR` for your change.
 
+### Adding a "what's new" post
+
+"What's new", sometimes referred to as "NR1 Announcements", provides a way to communicate new features in New Relic. These posts are displayed [on the documentation site](https://docs.newrelic.com/whats-new) as well as within the product. To add a new post:
```

Comment:
I think it would be great to link to where this lives in New Relic One: https://one.newrelic.com/launcher/jerome.plg-whats-new-launcher?pane=eyJuZXJkbGV0SWQiOiJqZXJvbWUucGxnLXdoYXRzLW5ldyJ9

This is the primary way we expect people to see these what's new posts.

Reply:
I'd rewrite this first sentence: 

We use "What's new" posts, or NR1 announcements, to let people know about new features and updates in [New Relic One](https://one.newrelic.com/launcher/jerome.plg-whats-new-launcher?pane=eyJuZXJkbGV0SWQiOiJqZXJvbWUucGxnLXdoYXRzLW5ldyJ9). You can also find What's new on our [Docs site](https://docs.newrelic.com/whats-new).

Reply:
I was spooked when my comments disappeared, but I see they wandered over here :D

Reply:
Sorry, I should give you some space before I start working on these adjustments 😆